Makefile

```diff
@@ -176,12 +176,18 @@ GO_IMAGE_TAG=$(shell cat .go-version)
 BUILD_IMAGE=public.ecr.aws/docker/library/golang:$(GO_IMAGE_TAG)
 BASE_IMAGE=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:latest.2
 GO_RUNNER_IMAGE=public.ecr.aws/eks-distro/kubernetes/go-runner:v0.15.0-eks-1-27-3
-.PHONY: docker-buildx
-docker-buildx: test
+.PHONY: docker-buildx-and-test
+docker-buildx-and-test: test
```

Comment:
I think there is no need to create another target just for different name. Adding test to docker-buildx is fine for me.